test/e2e/resources/kmeans_processingjob.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   processingInputs:
     - inputName: mnist_tar
       s3Input:
-        s3URI: s3://sagemaker-sample-data-$AWS_REGION/algorithms/kmeans/mnist/mnist.pkl.gz
+        s3URI: s3://$SAGEMAKER_DATA_BUCKET/algorithms/kmeans/mnist/mnist.pkl.gz
```

Comment:
can you please move this under `$SAGEMAKER_DATA_BUCKET/sagemaker/processing/` in source bucket

Reply:
s3 URI changed, file moved to new location in S3